src/main/resources/main.css

```diff
@@ -489,181 +493,26 @@ VBox.spaced, HBox.spaced {
 .labeled-link > .label {
     -fx-font-weight: bold;
 }
-
```

Comment:
why did you delete this? I think this css is important. The identity dropdown looks completely broken and I can't use the QR-Code feature anymore